lib/netdev_stdlib/version.rb

```diff
@@ -1,3 +1,3 @@
 module NetdevStdlib
-  VERSION = '0.10.0'
+  VERSION = '0.12.0'
```

Comment:
I'll pull out for the release PR.  It was missed in the last release, hence the double jump.